### ISSUE TITLE
Allow log file splitting to a maximum size (.csv)

### ISSUE THREAD

**Issue by boondoggler** [jtpereyda/boofuzz#530]
Forgive me if I have missed this in the help. Specifically for .csv (could be ported to text)

Large 1GB+ .csv files can be cumbersome to open using the usual suspects (I'm using a pretty capable 32GB RAM/3Ghz+ CPU desktop to view). 

Can a maximum log file size be set? I realise then you end up with numerous .csv log files - but it makes viewing easier.

**Comment by SR4ven** [comment]
Good point, splitting log files is currently not implemented but would indeed be quite handy.
